src/goSuggest.ts

```diff
@@ -274,7 +274,7 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider,
 						return reject();
 					}
 					let results = <[number, GoCodeSuggestion[]]>JSON.parse(stdout.toString());
-					let suggestions = [];
+					let suggestions: any[] = [];
```

Comment:
>any [](start = 22, length = 3)

suggestion seems to be of type `ExtendedCompletionItem` ?

Reply:
vscode.CompletionItem did the trick